src/scripts/main.js

```diff
@@ -1,7 +1,38 @@
 'use strict';
 
-const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+function pushNotification(posTop, posRight, title, description, type) {
+  const notification = document.createElement('div');
+
+  notification.className = `notification ${type}`;
+  notification.dataset.qa = 'notification';
+
+  notification.innerHTML = `
+    <h2 class="title">
```

Comment:
use letterSpacing for h2 and your cards will stand good, now they look not great